cetmix_tower_server/models/__init__.py

```
@@ -24,6 +24,8 @@
 from . import cx_tower_server_log
 from . import cx_tower_server_template
 from . import cx_tower_shortcut
+from . import cx_tower_scheduled_task
```

Flake8 (7.2.0)

[error] '.cx_tower_scheduled_task' imported but unused

(F401)